### PR TITLE
images/fedora: Fix fedora/34/cloud networking issues

### PR DESCRIPTION
**Pull request by monstermunchkin**

### CODE REVIEW

File: images/fedora.yaml [hunk 1]
```diff
@@ -338,3 +338,17 @@ actions:
...
   types:
   - container
+
+- trigger: post-files
+  action: |-
+    #!/bin/sh
+    set -eux
+
+    # This fixes the networking issues
+    sed -ri 's#ProtectKernelTunables=.*#ProtectKernelTunables=yes#g' /etc/systemd/system/service.d/lxc.conf
```
Comment (stgraber) [inline images/fedora.yaml:348]: Why is this needed and how isn't that breaking networkd like it normally does?

Comment (monstermunchkin) [reply images/fedora.yaml:348]: No idea why this is needed, but it fixes all networking issues. It's not breaking networkd because cloud images use NetworkManager.

Comment (monstermunchkin) [reply images/fedora.yaml:348]: Actually, this causes some other issues. Marking it as a draft.
